servant-client/src/Servant/Common/Req.hs

```diff
@@ -109,7 +109,7 @@ reqToRequest req (BaseUrl reqScheme reqHost reqPort) =
                                          , uriRegName = reqHost
                                          , uriPort = ":" ++ show reqPort
                                          }
-                             , uriPath = reqPath req
+                             , uriPath = path ++ reqPath req
                              }
```

Comment:
Is there a reason why this doesn't re-use `showBaseUrl`? That would reduce some duplication.